### ISSUE TITLE
Consolidate all regex expression into a single file and test

### ISSUE THREAD

**Issue by sethkfman** [MetaMask/metamask-mobile#6684]
### **Description**

Currently the mobile application has regex in a number of different files. In order to manage them better and test their functionality they should be moved to a utils file and tested. 

### **Technical Details**

- Move all regex functions into a single urls file and comment
- Write tests for all regex and confirm their functionality

### **Acceptance Criteria**

-   No functional impact is made
-   Regex are fully tested

**Comment by tibi77** [comment]
that s quite an interesting challenge. Of course, I m referring to finding all the regular expressions
My initial instinct is to write a regex that will find all the regexes, but after a lot of googling I could not find something that will suffice. Even StackOverflow [failed](https://unix.stackexchange.com/questions/582717/use-grep-to-find-files-containing-regex-patterns) to provide something useful. My next instinct is to go with the way that the regexes are being used... So this is what I ve done; I m quite curious, @sethkfman if you think we got them all or if some of them might have fallen through the cracks. 
Also, unsure if there is an easier way to find them all that I did not see
